bcmath/bcmath.php

```diff
@@ -150,15 +151,34 @@ function bcpow (string $num, string $exponent, ?int $scale = 0): string
 function bcsqrt (string $num, ?int $scale)
 {}
 
+#[PhpStormStubsElementAvailable(to: '7.3')]
 /**
  * Set default scale parameter for all bc math functions
  * @link https://php.net/manual/en/function.bcscale.php
- * @param int|null $scale <p>
- * The scale factor. Since 7.3.0 can be omitted.
- * </p>
- * @return int|true <b>INT</b> since 7.3.0 and <b>TRUE</b> before.
+ * @param int $scale
+ * @return true
+ */
+function bcscale (int $scale): bool
```

Comment:
Let's change duplicated functions to `LanguageLevelAware` attribute and leave just one function

Reply:
I don't know how to handle new default value NULL via ```LanguageLevelAware``` :-/

Reply:
That's right, unfortunately, it's not possible to handle default value via that attribute, but at least return types in two functions `function bcscale (int $scale): bool` and `function bcscale (int $scale): int` could be handled and leave only function `function bcscale (?int $scale = null): int` marked with `#[PhpStormStubsElementAvailable('8.0')]`